osu.Game/Screens/OnlinePlay/Multiplayer/Match/Playlist/MultiplayerPlaylist.cs

```diff
@@ -70,6 +70,8 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
             DisplayMode.BindValueChanged(onDisplayModeChanged, true);
+            queueList.Items.BindCollectionChanged(
+                (_, __) => playlistTabControl.queueListCount.Value = queueList.Items.Count, true);
```

Comment:
This can be simplified by letting `QueueTabItem` listen for changes to the `queueList.Items` bindable directly rather than defining another bindable for that:
```csharp
public class MultiplayerPlaylistTabControl
{
    public readonly IBindableList<PlaylistItem> QueueItems = new BindableList<PlaylistItem>();
    
    ... (in QueueTabItem)
    
    QueueItems.BindCollectionChanged((_, __) => Text.Text = ..., true);
}

...

playlistTabControl.QueueItems.BindTarget = queueList.Items;
```